Formula/catboost-cli.rb

```diff
@@ -0,0 +1,40 @@
+class CatboostCli < Formula
+  desc "Fast, scalable, high performance Gradient Boosting on Decision Trees cli tool"
+  homepage "https://catboost.ai"
+  url "https://github.com/catboost/catboost/archive/refs/tags/v1.2.tar.gz"
+  sha256 "59aa2c1690fff4ac38696f850a951484c844f0f40a0d0ab3a5b5dc588b09ee9b"
+  license "Apache-2.0"
+  head "https://github.com/catboost/catboost.git"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "cmake" => :build
+  depends_on "conan@1" => :build
+  depends_on "ninja" => :build
+
+  def install
+    extra_cmake_args = [
+      "-S",
+      buildpath.to_s,
+      "-B",
+      "#{buildpath}/brew-build",
+      "-G",
+      "Ninja",
+      "-DCATBOOST_COMPONENTS=app",
+      "-DHAVE_CUDA=NO",
+      "-DCMAKE_TOOLCHAIN_FILE=#{buildpath}/build/toolchains/clang.toolchain",
+    ]
+    system "cmake", *extra_cmake_args, *std_cmake_args
+    system "ninja", "-C", "#{buildpath}/brew-build", "catboost"
+
+    bin.install "#{buildpath}/brew-build/catboost/app/catboost" => "catboost-#{version}"
+    bin.install_symlink "catboost-#{version}" => "catboost"
```

Comment:
I've added the version to the binary because when you ask `catboost` for its version, you get this:

```
Arc info:
    Branch: unknown-vcs-branch
    Commit: 0577215664901532860606512090082402431042
    Author: ordinal
    Summary: No VCS
```

So, no info about its version is reported.

Reply:
Can we pass the version information to the build somehow?

Reply:
I've changed the strategy to download the source code.

By cloning the git repository, the VCS information is properly added:

```
Git info:
    Commit: d03b246cae23490dcf991cf822be110d6f818665
    Branch: tags/v1.2.1
    Author: akhropov <akhropov@yandex-team.com>
    Summary: Release 1.2.1..
    git-svn info:
    Last Changed Rev: 9293085
```

So now, I'm testing:

```ruby
assert_match %r{Branch: tags/v#{version}}, shell_output("#{bin}/catboost --version")
```